released/packages/coq-belgames/coq-belgames.1.1.0/opam

```diff
@@ -0,0 +1,43 @@
+# This file was generated from `meta.yml`, please do not edit manually.
+# Follow the instructions on https://github.com/coq-community/templates to regenerate.
+
+opam-version: "2.0"
+maintainer: "erik.martin-dorel@irit.fr"
+
+homepage: "https://github.com/pPomCo/belgames"
+dev-repo: "git+https://github.com/pPomCo/belgames.git"
+bug-reports: "https://github.com/pPomCo/belgames/issues"
+license: "MIT"
+
+synopsis: "BelGames: A Formal Theory of Games of Incomplete Information Based on Belief Functions in the Coq Proof Assistant"
+description: """
+We extend Bayesian games to the theory of belief functions. We
+obtain a more expressive class of games we refer to as BelGames. It
+makes it possible to better capture human behaviors with respect to
+lack of information.
+Next, we prove an extended version of the so-called
+Howson--Rosenthal's theorem, showing that BelGames can be turned
+into games of complete information, i.e., without any uncertainty."""
+
+build: [make "-j%{jobs}%"]
+install: [make "install"]
+depends: [
+  "coq" {(>= "8.14" & < "8.17~")}
+  "coq-mathcomp-algebra" {(>= "1.13" & < "1.16~")}
+]
+
+tags: [
+  "keyword:games of incomplete information"
+  "keyword:belief functions theory"
+  "keyword:mathcomp/ssreflect"
```

Comment:
```suggestion
  "keyword:mathematical components"
  "category:???"
```

No other package is using that keyword, but some of them are "`mathematical components`". Using it depends on whether you think your package is part of the Mathematical Components landscape (or just uses it).

Also having a category would be great. Not sure which one, however.

Reply:
Good point… better choose standard keywords/categories.
(We especially wanted to emphasize that we strongly rely on the mathcomp libraries so coq-belgames is compatible with them; given you suggest this is not enough to select this keyword, I'm going to remove it.)

Reply:
Should I also update the 1.0.0 keywords in this very same PR? or open another PR.

Reply:
As you wish. Both are fine with me.